gnovm/stdlibs/std/emit_event.go

```diff
@@ -0,0 +1,35 @@
+package std
+
+// ref: https://github.com/gnolang/gno/pull/853
+
+import (
+	"fmt"
+
+	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+	"github.com/gnolang/gno/tm2/pkg/sdk"
+)
+
+func X_emitEvent(m *gno.Machine, typ string, attrs []string) {
+	eventAttrs := make([]sdk.EventAttribute, len(attrs)/2)
+	pkgPath := CurrentRealmPath(m)
+
+	attrLen := len(attrs)
+	if attrLen%2 != 0 {
+		panic(fmt.Sprintf("attributes has an odd number of elements. current length: %d", attrLen))
```

Comment:
@notJoon I don't think we need panic here

Reply:
> Do we need to panic here?

Actually, I also think this panic unnecessary. it was just put in to prevent any unforeseen accident. I should modify it to just ignore it.